pkg/gallery/update.go

```diff
@@ -2,11 +2,63 @@ package gallery
 
 import (
 	"context"
+	"errors"
 	"fmt"
+	"time"
 
 	"github.com/stashapp/stash/pkg/models"
 )
 
+var ErrEmptyUpdater = errors.New("no fields have been set")
```

Comment:
Only related to this method definition change: https://github.com/stashapp/stash/pull/4423/files?diff=split&w=1#r1443579818
No change in behavior expected.